Tests/SwiftKuerySQLiteTests/TestSelect.swift

```diff
@@ -343,7 +343,7 @@ class TestSelect: XCTestCase {
                                     XCTAssertEqual(rows!.count, 1, "SELECT returned wrong number of rows: \(rows!.count) instead of 1")
                                     
                                     let s3 = Select(from: t)
-                                        .where(now() > t.date)
+                                        .where(now() >= t.date)
```

Comment:
Why does this logic need to change?

Reply:
More context:

```swift
                    let i1 = Insert(into: t, values: "now", Date(), Date(), Date())
                    executeQuery(query: i1, connection: connection) { result, rows in
                        // …

                            let i2 = Insert(into: t, values: "then", Date(timeIntervalSince1970: 0), Date(timeIntervalSince1970: 0), Date(timeIntervalSince1970: 0))
                            executeQuery(query: i2, connection: connection) { result, rows in
                                // …

                                    let s3 = Select(from: t)
                                        .where(now() >= t.date)
                                    executeQuery(query: s3, connection: connection) { result, rows in
                                        XCTAssertEqual(result.success, true, "SELECT failed")
                                        XCTAssertNotNil(result.asResultSet, "SELECT returned no rows")
                                        XCTAssertNotNil(rows, "SELECT returned no rows")
                                        // The assert below was the one that was failing
                                        XCTAssertEqual(rows!.count, 2, "SELECT returned wrong number of rows: \(rows!.count) instead of 2")
                                    }
```

The thing is that the tests run fast enough that there's unlikely to be a change of the current second between when the `i1` insert happens and the `s3` select happens, so `s3` was only selecting one row. I think this test used to succeed in the past because `now()` would insert the useless string "now" instead of an actual formatted datetime, and "now" is indeed "greater" than "2018-03-13 15:54:04" or any other similarly-formatted datetime according to SQLite.

Reply:
Oh, I get it. Makes sense